### DISCUSSION TITLE
Topic Note - How to use my Clipped Entry Template

### DISCUSSION THREAD

**Discussion by euzieg** [jgchristopher/obsidian-clipper#66] [Q&A]
My intention is to store clipper notes in my inbox folder with a dedicated template. 
**My settings:**
1. I deactivated "Daily-" and "Weekly Notes Entry" in the **Base** tab.
2. Entered the path to my template in the **Topic Note** tab under "Clipped Entry Template". 
3. Copied the "Template Example", which was generated with the link in the **Topic Note** tab, into the template file.
4. Additionally, I activated the "Store Clippings Per Domain" and named the storage location folder under **Advanced Settings** tab.

**My issue:**
The clippings are not using my defined template. Instead there's only a date direct followed by zulu time as header and appended content.

Why my template is not applied? What am I doing wrong?

**Comment by jgchristopher** [comment]
It sounds like you are using as expected. Let me test and get back to you. Can you share your template? Also, would you mind sharing screen shots of your config screen?

**Reply by euzieg** [reply]
Hi, here the additional information.

Obsidian version: v1.5.3

**My template:**
[clipper_template.md](https://github.com/jgchristopher/obsidian-clipper/files/14228437/clipper_template.md)

**Config:**
![Base_Screenshot 2024-02-10 081836](https://github.com/jgchristopher/obsidian-clipper/assets/145849789/b57db317-bf02-46fe-a36f-5acca9b8492b)
![Topic_Screenshot 2024-02-10 081904](https://github.com/jgchristopher/obsidian-clipper/assets/145849789/cb904b0b-185f-49df-8e9e-1ea7366ce973)
![Browser_Screenshot 2024-02-10 082032](https://github.com/jgchristopher/obsidian-clipper/assets/145849789/1ff837c9-3311-411e-9a8f-f9733146776c)
![Advanced_Screenshot 2024-02-10 081945](https://github.com/jgchristopher/obsidian-clipper/assets/145849789/dae4e5db-b8c0-4b56-8043-e528b4d904fb)

**My result looks like that:**
![Result_Screenshot 2024-02-10 082217](https://github.com/jgchristopher/obsidian-clipper/assets/145849789/9aa21810-6993-4746-919c-ffe1cf0aa9dd)
